UnityProject/Assets/Scripts/Systems/Antagonists/Antags/Changeling/ChangelingMain.cs

```diff
@@ -0,0 +1,626 @@
+using Mirror;
+using Newtonsoft.Json;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using Systems.Character;
+using UnityEngine;
+using Util;
+
+namespace Changeling
+{
+	[DisallowMultipleComponent]
+	public class ChangelingMain : NetworkBehaviour
```

Comment:
What Prefab is the script on?

Reply:
On prefab changeling (Assets\Prefabs\Changeling\ChangelingMain.prefab)

Reply:
ChangelingMain.prefab Is that a modification of the player prefab?

Reply:
No, it`s just prefab with changeling main script placed. That spawned when player is changeling on player game object as parent

Reply:
humm, would be best to have this functionality on the brain of the ling, So if they get surgically placed into a different body they still have the functionality